crates/ruff_python_formatter/resources/test/fixtures/ruff/statement/top_level.pyi

```diff
@@ -76,3 +76,9 @@ class Test2(A):
 def b(): ...
 # comment
 def c(): ...
+
+class EllipsisWithComment:
+    ...  # comment
+
```

Comment:
Could you add an example with a leading comment too

```
class EllipsisWithLeadingComment:
	# leading
	... 
```

and maybe another with two trailing comments

```
class EllispsisWithMultipleTrailing: # trailing class comment
	... # trailing ellipsis comment
```

Reply:
Black fails to format that first example. What should ruff do?

Reply:
We should keep the first example as is (not collapse the `...`).